src/frontend/.dockerignore

```diff
@@ -0,0 +1,9 @@
+node_modules
+ai-reviewer-admin/node_modules
+efiling-demo/node_modules
+efiling-frontend/node_modules
+build
+.dockerignore
+Dockerfile
+Dockerfile.prod
+.git
```

Comment:
Based on my research, this .dockerignore file is in the wrong directory.  I think this file can be removed.

https://medium.com/@LihauTan/took-me-hours-to-realise-why-docker-build-ignores-my-dockerignore-and-this-is-what-ive-learned-2f87c770ea9c

From what I understand, the file must be located **at the root directory of the context**.  For us, this means placing a file like this in each of the UI subfolders.  For this project, only in src/frontend/ai-reviewer-admin